cypress/support/cypress-commands/commands.js

```diff
@@ -688,6 +689,16 @@ Cypress.Commands.add('launchApp', (appType, appCallSign) => {
     };
     requestMap.params.intent.data = data;
   }
+  if (Cypress.env(CONSTANTS.TEST_TYPE).toLowerCase() == CONSTANTS.USERINTERESTPROVIDER) {
+    data = {
+      query: JSON.stringify({
+        params: {
+          [CONSTANTS.REGISTERPROVIDER]: false,
+        },
+      }),
+    };
+    requestMap.params.intent.data = data;
+  }
 
```

Comment:
Having if/else conditions for a single type of Test type or even for mf, is not a good practice

Reply:
Also proper comment to be added